README.md

```diff
@@ -68,6 +68,8 @@ docker-compose up -d
 poetry run createdb
 ```
 
+Of course you can also use an existing database instance, simply configure it in the `.env` file in the project root.
```

Comment:
I didn't realize that the create-script already populated an .env file with the db credentials, so maybe make this explicit in the docs.